src/gwc/src/test/java/org/geoserver/gwc/HttpRequestRecorderCallback.java

```diff
@@ -0,0 +1,32 @@
+package org.geoserver.gwc;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+import org.geoserver.ows.AbstractDispatcherCallback;
+import org.geoserver.ows.Request;
+
+public class HttpRequestRecorderCallback extends AbstractDispatcherCallback {
```

Comment:
Could be made ``final``, I know this is a tests support class, but I would add some minor comment explaining what is going on:

```
Integration tests support class that keeps track of the requests send to the dispatcher.
```

Reply:
No problem